hermes-common/src/main/java/pl/allegro/tech/hermes/common/metric/MetricsFacade.java

```diff
@@ -26,5 +30,13 @@ public SubscriptionMetrics subscriptionMetrics() {
     public ConsumerMetrics consumers() {
         return consumerMetrics;
     }
+
+    public PersistentBufferMetrics persistentBufferMetrics() {
```

Comment:
Let's agree on a convention for naming these methods. Currently, some of them are named `somethingMetrics`, while others are simply named `something`. For the sake of conciseness, I'd vote for the latter pattern.

Reply:
Agree, lets stick to names without `Metrics` suffix